packages/mds-agency/api.ts

```diff
@@ -1154,8 +1165,8 @@ function api(app: express.Express): express.Express {
         return map
       }, eventSeed)
       const telemetrySeed: { [s: string]: Telemetry } = {}
-      const telemetryMap = telemetry.reduce((map, telemetry) => {
-        map[telemetry.device_id] = telemetry
+      const telemetryMap = telemetry.reduce((map, t) => {
+        map[t.device_id] = t
```

Comment:
Could this be refactored using `Object.assign()`? Something like:
```typescript
const telemetryMap = telemetry.reduce((map, t) => {
   return Object.assign(map, { [t.device_id]: t })
}
```

Reply:
Yes. And in fact, we do this so often that we should create a helper function.